batchflow/models/tf/encoder_decoder.py

```diff
@@ -121,15 +130,11 @@ def default_config(cls):
 
         config['body/decoder'] = dict(skip=True, num_stages=None, factor=None)
         config['body/decoder/upsample'] = dict(layout='tna')
-        config['body/decoder/blocks'] = dict(base=cls.block, combine_op='softsum')
-        config['head'] = dict(layout='c', kernel_size=1)
-        return config
+        config['body/decoder/blocks'] = dict(base=cls.block, combine_op='concat')
 
-    def build_config(self, names=None):
-        config = super().build_config(names)
-        config['head/targets'] = self.get_from_attr('targets')
-        return config
+        config['head/layout'] = None
```

Comment:
is this line necessary?

Reply:
This is to explicitly show what the default parameter is